libcontainer/cgroups/utils.go

```diff
@@ -291,3 +293,36 @@ func GetHugePageSize() ([]string, error) {
 
 	return pageSizes, nil
 }
+
+func CheckSubsystem(path string, cgroup *configs.Cgroup, subsystem string) {
+	switch subsystem {
+	case "memory":
+		if cgroup.MemorySwap > 0 {
+			if !PathExists(filepath.Join(path, "memory.memsw.limit_in_bytes")) {
+				logrus.Warn("Your kernel does not support swap memory limit.")
```

Comment:
What about,
logrus.Warn("Your kernel does not support swap memory limit. Limitation discarded.")